packages/functional-tests/playwright.config.ts

```diff
@@ -13,23 +13,29 @@ const DEBUG = !!process.env.DEBUG;
 const SLOWMO = parseInt(process.env.PLAYWRIGHT_SLOWMO || '0');
 const NUM_WORKERS = parseInt(process.env.PLAYWRIGHT_WORKERS || '16');
 
-let retries = 0,
-  workers = NUM_WORKERS || 2,
+let workers = NUM_WORKERS || 2,
   maxFailures = 0;
 if (CI) {
-  // Overall maxFailures is now dependent on the number of retries, workers
-  retries = 3;
+  // Overall maxFailures is dependent on the number of workers
   workers = 2;
-  maxFailures = retries * workers * 2;
+  maxFailures = workers * 2;
 }
 
 export default defineConfig<PlaywrightTestConfig<TestOptions, WorkerOptions>>({
   outputDir: path.resolve(__dirname, '../../artifacts/functional'),
-  forbidOnly: CI,
-  retries,
+
+  // Look for test files in the "tests" directory, relative to this configuration file.
   testDir: 'tests',
+
   // Run all tests in parallel.
   fullyParallel: true,
+
+  // Fail the build on CI if you accidentally left test.only in the source code.
+  forbidOnly: CI,
+
+  // Retry on CI only.
+  retries: CI ? 1 : 0,
```

Comment:
Per https://playwright.dev/docs/test-retries#retries, we could also set this on a per test/suite level using inline `test.describe.configure({ retries: <number> })` if we know certain tests are more likely to need retries. (ie: disable globally, enable selectively).

Also, do we get the number of `retries` in the JUnit results or CircleCI? Wonder if it'd be a good thing to add to your dashboard.

Reply:
I don't think retries are accounted for in the JUnit report unfortunately, but it would be a very cool stat to have!

Reply:
Well, now you're making me question whether we need to see if there is a better JUnit custom reporter...

We'd have to see if https://github.com/Xray-App/playwright-junit-reporter is any better (https://www.npmjs.com/search?q=playwright%20junit didn't show a lot of promising alternatives).

Also, if we _did_ find a better JUnit reporter (or write our own, which, "gross"), see if we could include `file=` and possibly be able to use the CircleCI version of `--split-by=timings`.

Reply:
I was wrong:
<img width="1300" alt="image" src="https://github.com/mozilla/fxa/assets/3422688/0f1eb408-286a-4937-9211-2a238f5117db">